tests/Rust/tests/src/ExtInteropTests.rs

```diff
@@ -0,0 +1,32 @@
+pub mod ExtInteropTests {
```

Comment:
Perhaps change this to `InteropTests` too.

Reply:
OK, makes sense. But then, why not move those tests there, in InteropTests?

Reply:
The interop tests are written in F#, they are intended to test interop from the point of view of an F# developer.

The rust interop tests are written in Rust, because they are testing the rust experience of consuming F# generated types (currently just collections). I am not sure it makes sense to try and force these through with F#, even though it might be possible.

With that being said, I am not sure there is an easy way to merge both into 1 file?